From 35b2ec4949c9d1620551efde6e3a3d46323899de Mon Sep 17 00:00:00 2001
From: Yegor Yefremov <yegorslists@googlemail.com>
Date: Thu, 11 Jan 2018 10:13:18 +0100
Subject: [PATCH 2/2] Rework runtime dependencies

Add python version requirement to Python2 only packages like
ipaddress and enum34 and bump h2 package version.

Signed-off-by: Yegor Yefremov <yegorslists@googlemail.com>
---
 requirements-min.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/requirements-min.txt b/requirements-min.txt
index 83820cd1..5cbf35c4 100644
--- a/requirements-min.txt
+++ b/requirements-min.txt
@@ -2,7 +2,7 @@ click>=6.6
 setuptools>=28.3.0
 zope.interface>=4.3.2
 Twisted>=17.1.0
-h2<3.0.0
+h2>=3.0.1
 priority>=1.3.0
 txaio>=2.8.0
 autobahn>=17.6.2
@@ -33,5 +33,5 @@ bitstring>=3.1.5
 attrs>=16.2.0
 incremental>=16.10.1
 constantly>=15.1.0
-enum34>=1.1.6
-ipaddress>=1.0.18
+enum34>=1.1.6; python_version < '3.4'
+ipaddress>=1.0.18; python_version < '3.3'
-- 
2.11.0

